### PR TITLE
UX: Remove hardcoded "x" from tag count, use :before instead

### PR DESCRIPTION
**Pull request by carsick**
Allow customization for the count symbol (x) in list items like `Item x 4` with the :before pseudo-element

### CODE REVIEW

File: app/assets/javascripts/discourse/app/lib/render-tag.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ export function defaultRenderTag(tag, params) {
     ">";
 
   if (params.count) {
-    val += " <span class='discourse-tag-count'>x" + params.count + "</span>";
+    val += " <span class='discourse-tag-count'>" + params.count + "</span>";
```
Comment (ZogStriP) [inline app/assets/javascripts/discourse/app/lib/render-tag.js:65]: Should we be using the `tag-count` class here instead of the `discourse-tag-count`? 🤔

**Comment by ZogStriP** [comment]
@carsick I'm curious, what kind of customization are you looking for?

**Comment by carsick** [comment]
> @carsick I'm curious, what kind of customization are you looking for?

I was looking to display `Item` + `count` without the "x" symbol. I believe the "x" is an opinionated counter style; having it as a pseudo-element would better scale and accommodate diverse brand guidelines for typesetting.

**Comment by jbrw** [comment]
It looks like the failing frontend tests have the 'x' hard-coded in. Assuming we like the changes made by @carsick (I do, FWIW), we'll need to have the tests updated too.

File: app/assets/javascripts/discourse/app/lib/render-tag.js (hunk 1 above)
Comment (ZogStriP) [inline app/assets/javascripts/discourse/app/lib/render-tag.js:65]:
```suggestion
    val += ` <span class='discourse-tag-count'>${params.count}</span>`;
```

**Review by ZogStriP — APPROVED** [review]
LGTM though as @jbrw said, the failing specs seem genuine.

**Comment by ZogStriP** [comment]
@jbrw / @jordanvidrine any updates on this one?

**Pull request closed without merging**